packages/manager/modules/hub/src/components/billing-summary/index.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import ngOvhHttp from '@ovh-ux/ng-ovh-http';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import component from './component';
```

Comment:
Sort the imports please :)

Reply:
@ganeshkumar1989, in this case, vendor imports should always be at top level, so the import of the component comes after, as above

Reply:
Yeah got it @JeremyDec . Thanks :)